src/definition/validator/validate.js

```diff
@@ -96,13 +98,9 @@ function validateProperty(property, validator) {
  */
 function getErrorLalel(type, fieldName, options = {}) {
     options = options || {};
-    const i18n = require('i18next-client');
-    if (!i18n) {
-        throw new DependencyException('Dependency not resolved: i18n.js');
-    }
     const translationKey = options.translationKey ? options.translationKey : `domain.validation.${type}`;
-    const opts = assign({fieldName: i18n.t(fieldName)}, options);
-    return i18n.t(translationKey, opts);
+    const opts = assign({fieldName: translate(fieldName)}, options);
+    return translate(translationKey, opts);
 }
 
 module.exports = validate;
```

Comment:
It would be nice as you change `require` into `import` to change this into `export default`